src/connectors/hits-per-page/__tests__/connectHitsPerPage-test.ts

```diff
@@ -19,10 +30,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits-per-pa
     });
 
     it('throws with empty items', () => {
+      const items: HitsPerPageConnectorParamsItem[] = [];
```

Comment:
TS is supposed to throw once you pass the options with a wrong type. Wdyt about not using explicit types in the tests?